docs/tutorials/dynamics_backend.rst

```diff
@@ -0,0 +1,566 @@
+
+    from qiskit_experiments.library.calibration.rough_frequency import RoughFrequencyCal
+    
+    # experiment for qubit 0
+    freq0_estimate = v0 + 0.5e7
+    frequencies = np.linspace(freq0_estimate -15e6, freq0_estimate + 15e6, 27)
+    spec0 = RoughFrequencyCal(0, cals, frequencies, backend=backend)
+    spec0.set_experiment_options(amp=0.005)
+    
+    # experiment for qubit 1
+    freq1_estimate = v1 + 1e7
+    frequencies = np.linspace(freq1_estimate -15e6, freq1_estimate + 15e6, 27)
+    spec1 = RoughFrequencyCal(1, cals, frequencies, backend=backend)
+    spec1.set_experiment_options(amp=0.005)
+
+Visualize the first circuit for qubit 0.
+
+.. jupyter-execute::
+
+    spec0.circuits()[0].draw(output="mpl")
+
+Run the spectroscopy experiments.
+
+.. jupyter-execute::
+
+    %%time
+    spec0_data = spec0.run().block_for_results()
+    spec1_data = spec1.run().block_for_results()
+
+
+Plot the simulated data for both qubits.
+
+.. jupyter-execute::
+
+    spec0_data.figure(0)
+
+.. jupyter-execute::
+
+    spec1_data.figure(0)
+
+5.4 Rough amplitude calibration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next, run a rough amplitude calibration for ``X`` and ``SX`` gates for both qubits. First, build the
+experiments.
+
+.. jupyter-execute::
+
+    from qiskit_experiments.library.calibration import RoughXSXAmplitudeCal
+    
+    # rabi experiments for qubit 0
+    rabi0 = RoughXSXAmplitudeCal(0, cals, backend=backend, amplitudes=np.linspace(-0.2, 0.2, 27))
+    
+    # rabi experiments for qubit 1
+    rabi1 = RoughXSXAmplitudeCal(1, cals, backend=backend, amplitudes=np.linspace(-0.2, 0.2, 27))
+
+Run the Rabi experiments.
+
+.. jupyter-execute::
+
+    %%time
+    rabi0_data = rabi0.run().block_for_results()
+    rabi1_data = rabi1.run().block_for_results()
+
+Plot the results.
+
+.. jupyter-execute::
+
+    rabi0_data.figure(0)
+
+.. jupyter-execute::
+
+    rabi1_data.figure(0)
+
+Observe the updated parameters for qubit 0.
+
+.. jupyter-execute::
+
+    pd.DataFrame(**cals.parameters_table(qubit_list=[0, ()], parameters="amp"))
+
+5.5 Rough Drag parameter calibration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run rough Drag parameter calibration for the ``X`` and ``SX`` gates. This follows the same procedure
+as above.
+
+.. jupyter-execute::
+
+    from qiskit_experiments.library.calibration import RoughDragCal
+    
+    cal_drag0 = RoughDragCal(0, cals, backend=backend, betas=np.linspace(-20, 20, 15))
+    cal_drag1 = RoughDragCal(1, cals, backend=backend, betas=np.linspace(-20, 20, 15))
+    
+    cal_drag0.set_experiment_options(reps=[3, 5, 7])
+    cal_drag1.set_experiment_options(reps=[3, 5, 7])
+    
+    cal_drag0.circuits()[5].draw(output="mpl")
+
+.. jupyter-execute::
+
+    %%time
+    drag0_data = cal_drag0.run().block_for_results()
+    drag1_data = cal_drag1.run().block_for_results()
+
+.. jupyter-execute::
+
+    drag0_data.figure(0)
+
+
+.. jupyter-execute::
+
+    drag1_data.figure(0)
+
+
+5.6 Fine amplitude calibration
```

Comment:
These calibrations are all really cool to see, but you don't necessarily have to go through all of them since they're already in the calibrations tutorial and you can just refer the user to them. But for the ones that you keep, it would be helpful to link to the experiments's API docs.

Reply:
You're right. I've deleted the frequency calibrations as well as the fine calibrations. I've kept the rough amp and rough drag cals. I could maybe even drop the rough drag cals - I think it should be enough for the `CrossResonanceHamiltonian` experiment to have amplitudes roughly calibrated. That being said I do like to see both amplitude and drag.